docs/service-mesh/10-02-no-access.md

```diff
@@ -36,8 +34,13 @@ If this solution doesn't work, you need to change the image of the Istio Ingress
     kubectl exec -ti -n istio-system $(kubectl get pod -l app=istio-ingressgateway -n istio-system -o name) -c istio-proxy -- netstat -lptnu
     ```
 
-4. If ports `80` and `443` are not used, restart the Pods of the Istio Ingress Gateway to force them to recreate their configuration:
+4. If ports `80` and `443` are not used, check logs of the Istio Ingress Gateway container `ingress-sds` for errors related to the certificates:
```

Comment:
```suggestion
4. If ports `80` and `443` are not used, check the logs of the Istio Ingress Gateway container for errors related to certificates. Run:
```

Reply:
You don't really have to give the container name in the description - we have it in the command ;)